model/database/registry.yaml

```diff
@@ -534,6 +534,28 @@ groups:
         stability: experimental
         brief: Cosmos DB sub status code.
         examples: [1000, 1002]
+      - id: db.cosmosdb.consistency_level
+        type:
+          members:
+            - id: strong
+              value: "Strong"
```

Comment:
we use language-agnostic snake_case in other enum, values (`db.cosmosdb.connection_mode`, `db.cosmosdb.operation_type`) - let's follow the same pattern here?

```suggestion
              value: "strong"
```

Reply:
it should match with what we support in cosmos db
![image](https://github.com/user-attachments/assets/ad99e1d9-c781-42b5-955b-dbfbff7d997a)

Reply:
ideally we should change it for connection mode also 
![image](https://github.com/user-attachments/assets/505126d4-a07b-4fbb-941a-f7c82b0815fa)

Reply:
in which language?

Reply:
- C#...it's from .net sdk....I believe same value should be there for other flavors of sdk

Reply:
The constant names are different in each language. 

That's what you have in Python

<img width="355" alt="image" src="https://github.com/user-attachments/assets/0c644917-50f0-4deb-82c6-4d294cc8a7c2">


That's Java 

<img width="509" alt="image" src="https://github.com/user-attachments/assets/318fb570-a2d3-4b11-9a9e-d14186c5230d">


That's REST API definition

<img width="380" alt="image" src="https://github.com/user-attachments/assets/03ffc5b3-a9b1-4c1b-b369-142afc754689">


I agree with the choice to follow REST API spec as it's already consistent across languages.

But not everything is described in the REST API - e.g. connection mode is the one that's not there. And it's not the same in Java in .NET

<img width="314" alt="image" src="https://github.com/user-attachments/assets/1d917a54-a80e-404e-98fc-683910f663ae">


------------------------

TL;DR:
If you want to use REST API PascalCase constants for consistency level that's ok. Please take into consideration the overall consistency of the telemetry.

Having `operation.name = query_items` and `consistency.level = BoundedStaleness` would look weird.

Reply:
I am fine with it actually, because I don't want to include string manipulation in SDK just to make casing consistent. Value of Consistency is same everywhere, and for connection_mode also, it will be as it is. May be we can mention it our documentation that, casing doesn't matter here.

Reply:
Resolving this one, but I'm going to keep the discussion for connetion_mode open